doc/development-context-aware-applications/how-to-update-and-query-context-information.md

```diff
@@ -66,101 +81,201 @@ it in the following way:
+            },
+            {
+              "name": "occupancyLevels",
+              "type": "PropertyValue",
+              "value": 0,
+              "metadatas":
+              [
+                {
+                  "name": "timestamp",
+                  "type": "",
+                  "value": ""
+                },
+                {
+                  "name": "name",
+                  "type": "",
+                  "value": "occupancyLevels"
+                }
+              ]
+            },
+            {
+              "name": "position",
+              "type": "coords",
+              "value": "42.8404625, -2.5123277",
+              "metadatas":
+              [
+                {
+                  "name": "location",
+                  "type": "string",
+                  "value": "WGS84"
+                }
+              ]
+            },
+            {
+              "name": "priceRange",
+              "type": "",
+              "value": 0
+            },
+            {
+              "name": "telephone",
+              "type": "",
+              "value": "945 400 868"
+            }
+          ]
         },
+        "statusCode":
         {
-          "name": "cousine_type",
-          "type": "string",
-          "value": "french"
+            "code": "200",
+            "reasonPhrase": "OK"
         }
-        ],
-        "id": "LeBistro",
-        "isPattern": "false",
-        "type": "Restaurant"
-      },
-      "statusCode": {
-        "code": "200",
-        "reasonPhrase": "OK"
-      }
     }
 
+
+
 Alternatively, if you want to get the **attributes as a key map instead
 of as a vector**, you can use the attributesFormat parameter, in the
 following way
 
-    GET <cb_host>:<cb_port>/v1/contextEntities/type/Restaurant/id/LeBistro?attributesFormat=object
-
-    //getting a JSON response such as the following one:
+    GET <cb_host>:<cb_port>/v1/contextEntities/type/Restaurant/id/Elizalde?attributesFormat=object    
     {
-      "contextElement": {
-        "attributes": {
-          "name": {     
-            "type": "string",
-            "value": "Le Bistro"
-          },
-          "average_scoring": {
-            "type": "float",
-            "value": "4.2"
+      "contextElement" : {
+        "type" : "Restaurant",
+        "isPattern" : "false",
+        "id" : "Elizalde",
+        "attributes" : {
+          "address" : {
+            "type" : "",
+            "value" : {
+              "type" : "postalAddress",
```

Comment:
We are using the last docker image available at Docker Hub [link](https://hub.docker.com/r/fiware/tutorials.tourguide-app/builds/bdj2dbodbjmermzzxqbokj4/) . It seems to be up to date with the develop branch. We will rebuild and check it again